### ISSUE TITLE
Update user manual: Affordances

### ISSUE THREAD

**Issue by AnitaBats** [audacity/audacity#1280]

**Comment by petersampsonaudacity** [comment]
Note that there are an awful lot. of images of tracks in the manual which ideally should be updated to show the affordances drag-bar.

Including of course the "Amazing-Imagemap" on the front page - which provides a graphical "index" to content in the manual.

**Comment by petersampsonaudacity** [comment]
1_ I have added P1s in the manual for pages that I believe really must be changed - images and text.

2) There are other pages with track images that ideally should be changed.

3) I have added a placeholder section in the What's new for this new feature.

4) You really cannot call these thingamebobs/Dingsbums "Affordances" in a user-facing document like the Manual, it will totally close users:
a) it is not meaningful,
b) it is not accurate.  

@Tantacrul favors "**Clip-handles**" and I am minded to agree.  

I've found myself writing in the Manual "**Clip-handle drag bars**" for greater clarity.

**Comment by LWinterberg** [comment]
(this is blocked by #1446. Also by #1182 to some extend)

**Comment by LWinterberg** [comment]
As far as affordances go: 
- [ ] https://alphamanual.audacityteam.org/man/Sync-Locked_Track_Groups_-_Time_Shifting still needs to be done (after I properly understand what's happening there)
- [ ] https://alphamanual.audacityteam.org/man/Spectrogram_View may get some more images, but it's not that urgent
- [ ] https://alphamanual.audacityteam.org/man/Main_Page needs to get updated, and it seems like this can be done relatively painlessly. however, this is blocked by the design of the spectral editing tool/removal of the time shift tool button/...
- [ ] https://alphamanual.audacityteam.org/man/Time_Shift_Tool can get deleted (https://alphamanual.audacityteam.org/man/Special:WhatLinksHere/Time_Shift_Tool needs to be unlinked first)
- [ ] https://alphamanual.audacityteam.org/man/Editing,_resizing_and_moving_Labels would be nice-to-have for clip handles, but the focus is on labels which didn't change much in appearance

**Comment by petersampsonaudacity** [comment]
>     * https://alphamanual.audacityteam.org/man/Spectrogram_View may get some more images, but it's not that urgent

I trimmed the Timeline off of three images - which should remove the need to update these for the clip-handles.

**Comment by petersampsonaudacity** [comment]
@LWinterberg   I just added a P1 to this page
https://alphamanual.audacityteam.org/man/Audacity_Tracks_and_Clips#Moving_Clips

As the page on Time Shift tool is scheduled to be removed - this page needs to talk about the ability to shift audio to  before T-0 - i.e. audio in negative time.

**Comment by petersampsonaudacity** [comment]
@LWinterberg  I also added a P1 for updated images to this page:
https://alphamanual.audacityteam.org/man/Edit_Menu:_Clip_Boundaries

As this page is all about clips I think it really should be updated.

**Comment by petersampsonaudacity** [comment]
I also added P1s for

1.  https://alphamanual.audacityteam.org/man/Select_Menu:_Clip_Boundaries
2. https://alphamanual.audacityteam.org/man/Crossfade_Clips
3. https://alphamanual.audacityteam.org/man/Edit_Menu:_Remove_Special
4. https://alphamanual.audacityteam.org/m/index.php?title=Boundary_Snap_Guides&action=edit

As these all pay some attention to working with clips.

**Comment by petersampsonaudacity** [comment]
And this one

5. https://alphamanual.audacityteam.org/man/Keyboard_Shortcut_Reference#Extra:_Tools

**Comment by petersampsonaudacity** [comment]
Today I also changed several references to using the Time Shift tool - switching that to using the Clip-handle drag bars.

**Comment by LWinterberg** [comment]
Priorities have shifted, I'm considering the state of affordances in an acceptable state for now and will prioritize #1699 instead, as well as the new manual. I'm closing this even though some of these pages still haven't been updated, as the new system is scheduled for 3.2 anyway.
